lottie-swift/src/Private/LayerContainers/CompLayers/CompositionLayer.swift

```diff
@@ -143,7 +143,7 @@ class CompositionLayer: CALayer, KeypathSearchable {
   }
   
   func updateRenderScale() {
-    /// To be overridden by subclass
```

Comment:
What is this change for? It looks like at least some of the subclasses set `self.contentsScale = self.renderScale` in their `override func updateRenderScale()` implementations -- is there a reason this needs to be implemented in both places?

Reply:
Oh interesting, is this just a revert of #1328? If so then this change makes sense